src/styles/permissions.css

```diff
@@ -1,107 +1,132 @@
 /* Permissions */
 .sto-perm-list-title {
   margin-top: 0;
+  display: flex;
+  flex-direction: row;
+}
+
+.sto-perm-list-title p {
+  margin: 0;
+}
+
+.sto-perm-list-icon {
+  margin: 0 .5rem 0 0;
 }
 
 .sto-perm-list {
-    padding: 0;
-    list-style: none;
+  margin: 0 0 1rem calc(24px + .5rem);
+  padding: 0;
+  list-style: none;
 }
 
 .sto-perm-list li {
-    list-style: none;
-    display: flex;
-    margin: 1em 0
-}
-.sto-perm-list li:before {
-    content: '';
-    display: inline-block;
-    width: 1em;
-    height: 1em;
-    margin-right: .42em;
-    background-image: url('../assets/icons/permissions/icon-fallback.svg');
-    background-position: center center;
-    background-repeat: no-repeat;
-    background-size: contain;
-    vertical-align: -5em;
-    font-size: 3em
+  list-style: none;
+  margin: .5rem 0
 }
 
 .sto-perm-label {
   display: flex;
   flex-direction: column;
-  justify-content: center;
-  line-height: 1.3em;
-  font-size: 1.125em;
+  justify-content: left;
+  line-height: 1.5;
+  font-size: 16px;
 }
 
 .sto-perm-description {
   color: var(--cool-grey);
   font-size: .7em;
 }
 
-.sto-perm-list li[data-doctype^="io.cozy.settings"]:before {
-    background-image: url('../assets/icons/permissions/icon-settings.svg');
+.sto-perm-description-detail p {
+  margin: 0 0 0 1.5rem;
+  line-height: 1rem;
+  font-size: 12px;
+}
+
+.sto-perm-description-detail a {
+  color: var(--cool-grey);
+}
+
+.sto-perm-title {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+}
+
+.sto-perm-title:before {
+  content: '';
+  display: inline-block;
+  width: 1rem;
+  height: 1rem;
+  margin-right: .5rem;
+  background-image: url('../assets/icons/permissions/icon-fallback.svg');
+  background-position: top center;
+  background-repeat: no-repeat;
+  background-size: contain;
+}
+
+.sto-perm-title[data-doctype^="io.cozy.settings"]:before {
```

Comment:
In the future, I think we should not handle this icon with CSS and use a dedicated component.
```jsx
<DoctypeIcon doctype="io.cozy.settings" />
```